randovania/layout/lib/teleporters.py

```diff
@@ -39,6 +48,9 @@ class TeleporterShuffleMode(BitPackEnum, Enum):
 enum_lib.add_per_enum_field(TeleporterShuffleMode, "description", {
     TeleporterShuffleMode.VANILLA:
         "all elevators are connected to where they do in the original game.",
+    TeleporterShuffleMode.ECHOES_SHUFFLED:
+        "keeps Temple Grounds in place, shuffling the locations of all other areas with each other."
```

Comment:
We should be calling these "regions" or "worlds" imo, not "areas". There's a few other instances of this wording throughout the tab as well that we oughta fix while we're in here